game-service/src/main/java/org/libertybikes/game/core/GameRound.java

```diff
@@ -544,6 +535,33 @@ private void log(String msg) {
         public void gameEnding();
     }
 
+    private class Starter implements Runnable {
+
+        GameRound round;
+
+        public Starter(GameRound round) {
+            this.round = round;
```

Comment:
since this is an inner class of of `GameRound`, passing in the `GameRound` instance isn't necessary. We can just access it by doing `GameRound.this`